crates/cli/src/common_args.rs

```diff
@@ -0,0 +1,15 @@
+use clap::Arg;
+
+pub fn server() -> Arg {
+    Arg::new("server")
+        .long("server")
+        .short('s')
+        .help("The nickname, host name or URL of the server")
```

Comment:
if you do `.help` multiple times, does that latter one take precedence?

Reply:
I'm testing this rn

Reply:
confirmed this works as desired (see `Testing` section)

Reply:
Thanks, ideally it would just work and save us all a lot of suffering 😅